styles/main.css.map

```diff
@@ -0,0 +1 @@
ACA,4BACC,aAFF,8BADA,QACC,YAIA,eACC,gBACC,UAAA,CACA,UAON,QAAmC,iBAClC,WACC,gBAIC,aAAA,CACA,YANF,WACC,eACC,6BACC,aAHH,WAQC,eACC,4BACC,aAVH,WAaC,mBACC,UAAA,YACA,YAAA,YAGF,cACC,8BACC,kBACA,cAFD,6BAEE,QACA,YAKJ,QAAmC,iBAClC,WACC,kBACC,aAAA,CACA,gBC/SH,qBACE,gBAIF,KACE,iBAGF,OAAO,QACL,QAAS,EAAT,CACA,aAAA,CACA,gBAAA,CACA,WAAA,CACA,UAGF,WACE,YAAA,CACA,cAIA,CAAC,UAAW,IAAM,IAAM,IAAM,KAC9B,CAAC,UAAW,IAAM,IAAM,KACxB,CAAC,UAAW,IAAM,KAChB,yBAGF,CAAC,QAAS,IAAM,IAAM,IAAM,KAC5B,CAAC,QAAS,IAAM,IAAM,KACtB,CAAC,QAAS,IAAM,KACd,yBAGF,CAAC,MACC,kBAGF,CAAC,KAAK,QACJ,iBAAA,CACA,WAAA,CACA,YAAa,sBAAb,CACA,cAGF,CAAC,QAAS,EAAG,QAAQ,QACnB,WAGF,CAAC,YAAY,QACX,QAAS,QAGX,CAAC,QAAQ,QACP,QAAS,QAGX,CAAC,QAAQ,QACP,QAAS,QAGX,CAAC,UAAU,QACT,QAAS,QAGX,CAAC,QAAQ,QACP,QAAS,QAIb,IACE,IADF,IACM,IACF,mBAFJ,IAKE,GACE,IACE,gBAPN,IAKE,GAKE,EAAE,IACA,mBAKN,CAAE,MACF,EAAG,MAAM,EAAG,MAAM,EAAG,MAAM,EAAG,MAC5B,cAGF,EAAG,MAAM,EAAG,MAAM,EAAG,MAAM,EAAG,MAC5B,yBAGF,gBACE,gBADF,eAEE,SACE,eAAA,CACA,aAAA,CACA,oBCrGJ,OACI,eAAA,CACA,iBAAA,CACA,kCAAA,CACA,iBAJJ,MAMI,QACI,YAAA,CACA,iBAAA,CACA,wBATR,MAYI,KACI,eAAA,CACA,cAAA,CACA,aAAA,CACA,iBAAA,CACA,OAAA,CACA,UAAW,gBAAX,CACA,mCAAA,CACA,gCAAA,CACA,4BAGJ,MAAC,UACG,WAAA,CACA,iCAIR,UACI,eAAA,CACA,kBAAA,CACA,eCZJ,UACC,MACC,yBAFF,UAIC,IACC,aAAA,CACA,kBANF,UAQC,MACC,aAAA,CACA,yBAVF,UAYC,IACC,aAAA,CACA,iBAdF,UAgBC,IACC,aAAA,CACA,iBAlBF,UAoBC,KACC,aAAA,CACA,kBAtBF,UAwBC,KACC,aAAA,CACA,gBAAA,CACA,kBA3BF,UA6BC,KACC,aAAA,CACA,kBA/BF,UAiCC,KACC,aAAA,CACA,gBAAA,CACA,kBApCF,UAsCC,KACC,aAAA,CACA,yBAxCF,UA0CC,KACC,aAAA,CACA,kBA5CF,UA8CC,KACC,cA/CF,UAiDC,KACC,cAlDF,UAoDC,KACC,aAAA,CACA,yBAtDF,UAwDC,KACC,cAzDF,UA2DC,KACC,cA5DF,UA8DC,KACC,iBA/DF,UAiEC,KACC,cAlEF,UAoEC,KACC,cArEF,UAuEC,KACC,aAAA,CACA,iBAzEF,UA2EC,KACC,aAAA,CACA,iBA7EF,UA+EC,KACC,aAAA,CACA,iBAjFF,UAmFC,KACC,aAAA,CACA,iBArFF,UAuFC,KACC,aAAA,CACA,iBAzFF,UA2FC,KACC,aAAA,CACA,iBA7FF,UA+FC,IACC,cAhGF,UAkGC,IACC,cAnGF,UAqGC,KACC,cAtGF,UAwGC,KACC,cAzGF,UA2GC,KACC,aAAA,CACA,iBA7GF,UA+GC,KACC,cAhHF,UAkHC,KACC,aAAA,CACA,iBApHF,UAsHC,KACC,cAvHF,UAyHC,KACC,aAAA,CACA,iBA3HF,UA6HC,KACC,aAAA,CACA,iBA/HF,UAiIC,KACC,aAAA,CACA,iBAnIF,UAqIC,KACC,cAtIF,UAwIC,KACC,cAzIF,UA2IC,KACC,cA5IF,UA8IC,KACC,aAAA,CACA,iBAhJF,UAkJC,IACC,cAnJF,UAqJC,KACC,cAtJF,UAwJC,KACC,cAzJF,UA2JC,KACC,cA5JF,UA8JC,KACC,cA/JF,UAiKC,KACC,cAlKF,UAoKC,KACC,cArKF,UAuKC,KACC,cAxKF,UA0KC,KACC,cA3KF,UA6KC,KACC,cA9KF,UAgLC,KACC,cAjLF,UAmLC,KACC,cApLF,UAsLC,KACC,cAvLF,UAyLC,KACC,cA1LF,UA4LC,KACC,cA7LF,UA+LC,KACC,cAhMF,UAkMC,KACC,cAnMF,UAqMC,KACC,cAtMF,UAwMC,KACC,cAzMF,UA2MC,KACC,cA5MF,UA8MC,KACC,cCpOF,UACI,cADJ,UACkB,UACV,SAFR,UAII,OACI,UALR,UAQI,GACI,SAIR,YACI,EAAE,EAAG,QACD,QAAS,GAAT,CACA,oBAAA,CACA,iBCjBR,qBACE,cAAA,CACA,mBAEA,QAA+C,0BA+BjD,qBA9BI,cA8BJ,oBA5BI,IACE,gBA2BN,oBAzBI,IACE,kBAIJ,qBAfF,oBAeK,IAfL,oBAeW,IAfX,oBAeiB,IAfjB,oBAeuB,IAfvB,oBAe6B,IACzB,kBADF,oBAGE,gBAlBJ,oBAeK,GAGD,gBAlBJ,oBAeW,GAGP,gBAlBJ,oBAeiB,GAGb,gBAlBJ,oBAeuB,GAGnB,gBAlBJ,oBAe6B,GAGzB,gBACE,aAnBN,oBAuBE,EAAI,sBACF,cAAA,CACA,gBAAA,CACA,kBA1BJ,oBA6BE,KACE,WAAA,CACA,uBAAA,CACA","file":"main.css"}
```

Comment:
Can you remove this sourcemap? I think we had a sourcemap in the repo at one point, but we removed it because it only contains user-specific info. For example, this file contains paths on your local filesystem, so it wouldn't work for other people.